metrics/metrics/src/main/java/io/helidon/metrics/RegistryFactory.java

```diff
@@ -164,7 +164,7 @@ public void update(MetricsSettings metricsSettings) {
         });
     }
 
-    private synchronized void ensureBase() {
+    private void ensureBase() {
```

Comment:
Just a note that I'm leery of simply removing `synchronized`; maybe it's OK but definitely have another look just to be safe.  Guessing that the call to `accessMetricsSettings()` holds the lock internally.

Reply:
Yes, the lock is managed inside `accessMetricsSettings`.